android/expoview/src/main/java/host/exp/exponent/LauncherActivity.kt

```diff
@@ -0,0 +1,83 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+package host.exp.exponent
+
+import android.Manifest
+import android.app.Activity
+import android.app.ActivityManager.RecentTaskInfo
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.os.Build
+import android.os.Bundle
+import android.os.Handler
+import androidx.core.content.ContextCompat
+import host.exp.exponent.di.NativeModuleDepsProvider
+import host.exp.exponent.kernel.Kernel
+import host.exp.expoview.BuildConfig
+import javax.inject.Inject
+
+// This activity is transparent. It uses android:style/Theme.Translucent.NoTitleBar.
+// Calls finish() once it is done processing Intent.
+class LauncherActivity : Activity() {
+  @Inject
+  lateinit var kernel: Kernel
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    if (BuildConfig.DEBUG) {
```

Comment:
might be worth looking in the git history to see why the java code was written as host.exp.expoview.BuildConfig.DEBUG. could be an accident, could be intentional for a code modifying script

Reply:
https://github.com/expo/expo/commit/fdf094e253d5c4ea52e6be6bc59e83123742623e changed it. from `BuildConfig.DEBUG`. Looks like an accident as the rest of the PR doesn't do anything with code modification.